proms-resources/frontend/src/main/frontend/src/PromsView.jsx

```diff
@@ -0,0 +1,61 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useEffect, useState } from "react";
+
+import { Grid, Typography, withStyles } from "@material-ui/core";
+import questionnaireStyle from "../questionnaire/QuestionnaireStyle.jsx";
+import FormView from "./FormView.jsx";
```

Comment:
In aggregated-frontend, this source will end up in the top-level directory. Relative paths to sources from other modules must start with `./`.
```suggestion
import questionnaireStyle from "./questionnaire/QuestionnaireStyle.jsx";
import FormView from "./dataHomepage/FormView.jsx";
```